appservice/src/intTest/groovy/nu/yona/server/UserTest.groovy

```diff
@@ -59,7 +59,7 @@ class UserTest extends AbstractAppServiceIntegrationTest
 		// The below asserts check the path fragments. If one of these asserts fails, the Swagger spec needs to be updated too
 		john.postOpenAppEventUrl == baseUserUrl + "/devices/" + john.getRequestingDeviceId() + "/openApp"
 		john.buddiesUrl == baseUserUrl + "/buddies/"
-		john.goalsUrl == baseUserUrl + "/goals/"
+		john.goalsUrl.startsWith(baseUserUrl + "/goals/")
```

Comment:
Shouldn't we assert that the left hand side with stripped query string is equal to the right hand side?

Reply:
I followed your suggestion, as that makes it more clear what we are exactly trying to accomplish.